src/desktop/DeviceKeyProviderImpl.ts

```diff
@@ -7,64 +7,79 @@ import {defer} from "@tutao/tutanota-utils"
 import {base64ToKey, keyToBase64} from "@tutao/tutanota-crypto"
 // exported for testing
 export const SERVICE_NAME = "tutanota-vault"
-export const ACCOUNT_NAME = "tuta"
+
+export enum KeyAccountName {
+	DEVICE_KEY = "tuta",
+	CREDENTIALS_KEY = "credentials-device-lock-key"
+}
 
 export interface DesktopDeviceKeyProvider {
```

Comment:
We should rename this to just `DesktopKeyProvider` since it now provides more than just a device key

Reply:
I used KeyStoreFacade, since that name is used on android